docs/source/azure/examples/functionappopencors-compute.rst

```diff
@@ -0,0 +1,20 @@
+.. _azure_examples_function_app_cors:
+
+Function Apps - Find Functions with Open CORS Policy
+===============================================================
+
+Filter to select all Function Apps with open an CORS policy. An open CORS policy 
+here defined as a wildcard, `*`.
```

Comment:
Filter to select all Application Services (web applications, functions) with a Cross-Origin Resource Sharing (CORS) configuration set to allow all origins.